observability/distr_tracing/distr-tracing-rn.adoc

```diff
@@ -55,8 +60,47 @@ This update introduces the following bug fix:
 
 * Before this update, the {TempoOperator} failed when the `TempoStack` custom resource had the `spec.storage.tls.enabled` field set to `true` and used an Amazon S3 object store with the Security Token Service (STS) authentication. With this update, such a `TempoStack` custom resource configuration does not cause the {TempoOperator} to fail.
 
+[id="distr-tracing_3-5_tempo-release-notes_known-issues_{context}"]
+==== Known issues
+
+There is currently a known issue:
+
+* Currently, when the OpenShift tenancy mode is enabled, the ServiceAccount of the gateway component of a TempoStack or TempoMonolithic instance requires the TokenReview and SubjectAccessReview permissions for authorization.
++
+Workaround: Deploy the instance in a dedicated namespace, and carefully audit which users have the permission to read the secrets in this namespace.
+
+[id="distr-tracing_3-5-1_tempo-release-notes_{context}"]
+=== {TempoName} 3.5.1
+
+The {TempoName} 3.5.1 is a patch release.
+
+[id="distr-tracing_3-5-1_tempo-release-notes_cves_{context}"]
+==== CVEs
+
+The {TempoName} 3.5.1 patch release fixes the following CVEs:
+
+* link:https://access.redhat.com/security/cve/CVE-2025-2786[CVE-2025-2786]
+
+* link:https://access.redhat.com/security/cve/CVE-2025-2842[CVE-2025-2842]
+
+[id="distr-tracing_3-5-1_tempo-release-notes_breaking-changes_{context}"]
+==== Breaking changes
+
+The {TempoName} 3.5.1 update introduces the following breaking change:
+
+* With this update, for a user to create or modify a `TempoStack` or `TempoMonolithic` custom resource with enabled multi-tenancy, the user must have permissions to create a TokenReview and SubjectAccessReview.
+
+[id="distr-tracing_3-5-1_tempo-release-notes_known-issues_{context}"]
+==== Known issues
+
+There is currently a known issue:
+
+* Currently, when the OpenShift tenancy mode is enabled, the ServiceAccount of the gateway component of a TempoStack or TempoMonolithic instance requires the TokenReview and SubjectAccessReview permissions for authorization.
```

Comment:
```suggestion
* Currently, when the OpenShift tenancy mode is enabled, the `ServiceAccount` object of the gateway component of a `TempoStack` or `TempoMonolithic` instance requires the `TokenReview` and `SubjectAccessReview` permissions for authorization.
```

Reply:
:slightly_smiling_face: Alright. Thank you. Applied.